src/locales/en-US/volume.js

```diff
@@ -1,7 +1,6 @@
 export default {
-  'volume.name': 'name',
-  'volume.storageName': 'storage name',
-  'volume.accessMode': 'access mode',
+  'volume.storageName': 'storage Name',
+  'volume.accessMode': 'access Mode',
```

Comment:
Let's use lowercase, according to this comment.
https://github.com/linkernetworks/vortex-portal/pull/33#issuecomment-418613890

Reply:
keep as `access mode`,  `storage name`